package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-localstack",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
Let's add a short entry to the Change Log section in the README. 👍

Reply:
done and moved to minor version bump